conan/tools/build/cppstd.py

```diff
@@ -104,6 +118,52 @@ def supported_cppstd(conanfile, compiler=None, compiler_version=None):
     return None
 
 
+def _check_cppstd(conanfile, cppstd, comparator, comparator_error_string, gnu_extensions) -> bool:
```

Comment:
I would avoid passing the ``comparator_error_string``, such string can be deduced from ``comparator`` when necessary.

Reply:
Changed in [59dcb73](https://github.com/conan-io/conan/pull/11610/commits/59dcb73a9f882eee781353f11fdb7a6a9c897546)